.gitignore

```
@@ -1,3 +1,4 @@
 target/
+.bsp/
```

I'm pretty sure this only happens at the top-level, so it should be
```suggestion
/.bsp/
```

---

Should be fixed.